src/count/src/javascripts/widget.js

```diff
@@ -0,0 +1,111 @@
+const envConfig = require('./config.js');
+const oCommentUtilities = require('o-comment-utilities');
+const oCommentApi = require('o-comment-api');
+
+const Widget = function (rootEl, config) {
+	let widgetEl;
+	let commentCount;
+
+	try {
+		if (!rootEl) {
+			rootEl = document.body;
+		} else if (!(rootEl instanceof HTMLElement)) { // could throw exception in IE
+			rootEl = document.querySelector(rootEl);
+		}
+	} catch (e) {
+		let el;
+		if (typeof rootEl === 'string') {
+			el = document.querySelector(rootEl);
+		}
+
+		if (el) {
+			rootEl = el;
+		} else {
+			rootEl = document.body;
```

Comment:
I think this block is redundant now this count widget isn't being treated as a separate component? I.e. if `rootEl` is set to `document.body` here we'll get an error?

Reply:
👍 and done!